Language/Structure/Control Structure/else.adoc

```diff
@@ -57,15 +57,15 @@ Below is an extract from a code for temperature sensor system
 ----
 if (temperature >= 70)
 {
-  //Danger! Shut down the system
+  // Danger! Shut down the system
 }
-else if (temperature >= 60 && temperature < 70)
+else if (temperature >= 60)
 {
-  //Warning! User attention required
+  // Warning! Thorttle down, run the fans, and notify the user

```

Comment:
You introduced a typo here (should be spelled "Throttle"). I'm actually not convinced changing this comment provides any benefit, though I do very much agree with the change to a consistent comment style.